src/status_im2/contexts/chat/lightbox/zoomable_image/utils.cljs

```diff
@@ -1,31 +1,32 @@
 (ns status-im2.contexts.chat.lightbox.zoomable-image.utils
   (:require
     [clojure.string :as string]
+    [react-native.core :as rn]
     [react-native.navigation :as navigation]
     [react-native.orientation :as orientation]
     [react-native.platform :as platform]
     [react-native.reanimated :as reanimated]
     [reagent.core :as reagent]
-    [status-im2.contexts.chat.lightbox.zoomable-image.constants :as c]
+    [status-im2.contexts.chat.lightbox.zoomable-image.constants :as constants]
     [status-im2.contexts.chat.lightbox.animations :as anim]
     [utils.re-frame :as rf]))
 
 ;;; Helpers
 (defn center-x
   [{:keys [pinch-x pinch-x-start pan-x pan-x-start]} exit?]
-  (let [duration (if exit? 100 c/default-duration)]
-    (anim/animate pinch-x c/init-offset duration)
-    (anim/set-val pinch-x-start c/init-offset)
-    (anim/animate pan-x c/init-offset duration)
-    (anim/set-val pan-x-start c/init-offset)))
+  (let [duration (if exit? 100 constants/default-duration)]
```

Comment:
Just wondering, have we settled on using `constants` or `c`? I remember in the past there were discussions about this and now it's changing again from `c` to `constants`. I always thought we aligned `constants` was the accepted convention, but we merged as `c` anyway.

Reply:
It was a suggestion to use `c` for the local constants file and `constants` for the global one, but later on not everyone accepted it. So I am changing it back.

Reply:
Thanks @OmarBasem, so now it's always `constants`? Global or not? If that's the case we could open a separate PR updating the `lint-as` rule in `clj-kondo/config.edn`. The rule doesn't allow regexps, but we just have a handful of constants namespaces anyway, so it works well in practice if we hardcode the list.

Reply:
@ilmotta Yeah we can do that 👍